test/c_interface_test.cpp

```diff
@@ -26,7 +26,9 @@ using namespace spvtools;
 TEST(CInterface, DefaultConsumerNullDiagnosticForValidInput) {
   auto context = spvContextCreate(SPV_ENV_UNIVERSAL_1_1);
   const char input_text[] =
-      "OpCapability Shader\nOpMemoryModel Logical GLSL450";
+      "OpCapability Shader\n"
+      "OpCapability Linkage\n"
```

Comment:
It turns out that Vulkan does not permit Linkage capability. So this combination doesn't correspond to a real environment.  But that's ok.  A distinct Vulkan env check is yet to be coded.